templates/containers/chiselled/dotnet.html

```diff
@@ -0,0 +1,307 @@
+{% extends "containers/chiselled/_base_chiselled.html" %}
+
+{% block title %}Chiselled Ubuntu and .NET{% endblock title %}
+
+{% block meta_description %}Chiselled Ubuntu for .NET is a game-changing technology that delivers the best advantages of Distroless containers with Ubuntu's reliability and ease of use.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1fUG2SpbfdVDMAdbe7zfuzzvji4oWm-P18D85V2A3F8c/edit{% endblock meta_copydoc %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6 col-medium-3">
+      <h1 class="p-heading--bold p-heading--2">Migrate to .NET and <br class="u-hide--small"/>chiselled Ubuntu</h1>
+    </div>
+    <div class="col-6 col-medium-3">
+      <p class="p-block">Deliver reliable and secure applications with .NET and Ubuntu containers. Reduce memory footprint, lower subscription and network costs, and get faster start-up times. Open source and backed by Canonical security, stability, and quality assurance.</p>
+      <p>
+        <a href="/containers/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
+        <p class="p-heading--2 u-no-margin"><strong>20%</strong></p>
+        <p class="p-heading--4 u-no-margin">Better memory usage</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
+        <p class="p-heading--2 u-no-margin"><strong>40%</strong></p>
+        <p class="p-heading--4 u-no-margin">Better response time</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
+        <p class="p-heading--2 u-no-margin"><strong>25%</strong></p>
+        <p class="p-heading--4 u-no-margin">Fewer resources</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
+        <p class="p-heading--2 u-no-margin"><strong>50%</strong></p>
+        <p class="p-heading--4 u-no-margin">Less costs</p>
+      </div>
+    </div>
+    <p>Data is based on migration from servers to containers <br class="u-hide--small u-hide--medium"/>and from Windows to Ubuntu with chiselled Ubuntu.</p>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="p-rule" />
+    <div class="col-6 col-medium-3">
+      <h2>What is chiselled Ubuntu?</h2>
```

Comment:
```suggestion
      <h2>What is chiselled Ubuntu</h2>
```
There's no punctuation on these headings on the doc